nextmv/nextmv/options.py

```diff
@@ -942,6 +974,12 @@ def _description(self, option: Option) -> str:
         if isinstance(option, Option) and option.additional_attributes is not None:
             description += f" (additional attributes: {option.additional_attributes})"
 
+        if isinstance(option, Option) and option.control_type is not None:
+            description += f" (control type: {option.control_type})"
+
+        if isinstance(option, Option) and option.hidden_from is not None:
```

Comment:
Maybe we only add it if there actually are values defined and not on empty list? But just a thought. :blush: 

```suggestion
        if isinstance(option, Option) and option.hidden_from:
```